fsfuzz

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
On RHEL7 it works fine with either of them, so I can't tell, anyway I don't see any harm in this.